metadata-ingestion/tests/unit/fivetran/test_fivetran_config.py

```diff
@@ -0,0 +1,136 @@
+import pytest
+from pydantic import ValidationError
+
+from datahub.ingestion.source.fivetran.config import (
+    MAX_COLUMN_LINEAGE_PER_CONNECTOR_DEFAULT,
+    MAX_JOBS_PER_CONNECTOR_DEFAULT,
+    MAX_TABLE_LINEAGE_PER_CONNECTOR_DEFAULT,
+    FivetranLogConfig,
+    FivetranSourceConfig,
+    SnowflakeDestinationConfig,
+)
+
+
+class TestFivetranConfig:
+    """Test cases for Fivetran configuration with configurable limits."""
+
+    def test_fivetran_log_config_default_limits(self):
```

Comment:
Is this test meaningful?

Reply:
Well, it's meaningful.

I agree that the test is just covering pydantic field defaults ,and because of that it may be considered redundant or unnecessary.

Removing test file.